rosetta-ide/src/main/java/com/regnosys/rosetta/ide/server/RosettaLanguageServerImpl.java

```diff
@@ -170,4 +175,15 @@ protected SemanticTokens semanticTokensRange(SemanticTokensRangeParams params, C
 	public CompletableFuture<SemanticTokens> semanticTokensRange(SemanticTokensRangeParams params) {
 		return this.getRequestManager().runRead((cancelIndicator) -> this.semanticTokensRange(params, cancelIndicator));
 	}
+
+	@Override
+	public CompletableFuture<FormattingOptions> getDefaultFormattingOptions() {
+		try {
+			return CompletableFuture.completedFuture(formattingOptionsAdapter.readFormattingOptions(null));
+		} catch (IOException e) {
+			// should never happen, since null path always leads to default options being returned
+			e.printStackTrace();
+			return null;
```

Comment:
Return a `CompletableFuture.failedFuture(e)` in this case, so the client knows if anything wrong happened.